src/sentry/static/sentry/less/group-detail.less

```diff
@@ -945,13 +945,7 @@
     position: relative;
     top: 1px;
   }
-
-  .icon-question {
-    font-size: 15px;
-    position: relative;
-    top: 1px;
-  }
-
+  
   .in-at {
```

Comment:
These changes fix some content "popping" that happens when you toggle between original/minified code.